config/initializers/sidekiq.rb

```diff
@@ -40,6 +49,13 @@ def self.read_redis_config
   config.server_middleware do |chain|
     chain.add Sidekiq::Congestion::Limiter
   end
+
+  config.options[:concurrency] = ENV['SIDEKIQ_CONCURRENCY'].to_i || 5
```

Comment:
These can just go back to a checked in YAML template with ENV variable inject at runtime, like caesar db config, https://github.com/zooniverse/caesar/blob/e3778e0368356b2d5a40abdace37abcea53ab6df/config/database.yml#L4